src/tests/Interop/DllImportSearchPaths/DllImportSearchPathsTest.csproj

```diff
@@ -14,11 +14,20 @@
       <FileNameSuffix>-in-subdirectory</FileNameSuffix>
     </PropertyGroup>
     <ItemGroup>
-      <_FilesToCopy Include="$(OutDir)/$(TargetName).dll" />
       <_FilesToMove Include="$(OutDir)/libNativeLibrary.*" />
       <_FilesToMove Include="$(OutDir)/NativeLibrary.*" />
     </ItemGroup>
-    <Copy SourceFiles="@(_FilesToCopy)" DestinationFiles="@(_FilesToCopy -> '$(NativeLibrarySubdirectory)/%(Filename)%(Extension)')" />
     <Move SourceFiles="@(_FilesToMove)" DestinationFiles="@(_FilesToMove -> '$(NativeLibrarySubdirectory)/%(Filename)%(Extension)')" />
   </Target>
+
+  <Target Name="SetUpSubdirectoryManaged" AfterTargets="Build" Condition="'$(__SkipManaged)' != '1'">
```

Comment:
nit: rename the existing target to SetUpSubdirectoryNative to match?

Reply:
Done